### ISSUE TITLE
[Bug] Can't open the file when subtitle name containing ";"

### ISSUE THREAD

**Issue by SeanPhy** [wyzdwdz/assfonts#20]
When using the GUI, the file name cannot contain ";", see screenshot.
![Screenshot 2024-12-15 194623](https://github.com/user-attachments/assets/ce010188-3e98-4d41-9fd2-aeba91e639bf)

GUESS: ";" is already used to separate files.

**Comment by wyzdwdz** [comment]
Yeah, you are right. ";" is used for separating files. You can just work around by renaming your filename. Sorry recently I have no much time to fix this bug.
